sesame/src/main/scala/org/w3/banana/sesame/SesameOps.scala

```diff
@@ -167,7 +167,7 @@ class SesameOps extends RDFOps[Sesame] with SesameMGraphOps with DefaultURIOps[S
   def isomorphism(left: Sesame#Graph, right: Sesame#Graph): Boolean = {
     val leftNoContext = left.asScala.map(s => makeTriple(s.getSubject, s.getPredicate, s.getObject)).asJava
     val rightNoContext = right.asScala.map(s => makeTriple(s.getSubject, s.getPredicate, s.getObject)).asJava
-    ModelUtil.equals(leftNoContext, rightNoContext)
+    Models.isomorphic(leftNoContext, rightNoContext)
```

Comment:
@antonkulaga I think @betehess is asking if you checked that the two lines above actually are still correct independent of our tests that may not be complete, since those lines were introduced to fix an issue as explained in the javadoc above

```java
  /**
   * the new ModelUtil.equals changed its semantics. See
   * - https://openrdf.atlassian.net/browse/SES-1695
   * - https://groups.google.com/forum/#!topic/sesame-devel/CGFDn7mESLg/discussion
   */
```